### DISCUSSION TITLE
Search Query within GET Request

### DISCUSSION THREAD

**Discussion by extrafein** [Kovah/LinkAce#383] [General]
Hi,

is there the possibility to have search-parameters within a GET request.
Something like https://linkace.org/search?query=mysearchstring
This would make integration of apps like Alfred (OSX) possible.

Thank in advance.

Marcus

**Comment by Kovah** [comment]
Hi, thanks for your interest in LinkAce.
Searches via GET requests are currently not available, they must be done via POST.

**Comment by rinze** [comment]
Hi,

Are there any plans to implement this? It'd also help creating customized search commands on Firefox / Chrome. For example, for my Shaarli instance I currently have:

```
https://myshaarlidomain.org/?searchterm=%s
```

And I have that mapped to letter `b`, so I can open a new tab and do `b thistag`, and I'll go directly to the results page.

Thanks!
